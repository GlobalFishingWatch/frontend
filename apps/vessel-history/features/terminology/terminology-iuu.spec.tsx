import { Trans, useTranslation } from 'react-i18next'
import { render } from '@testing-library/react'

import TerminologyIuu from './terminology-iuu'

jest.mock('react-i18next')

describe('TerminologyIuu', () => {
  const mockUseTranslation: jest.Mock = useTranslation as jest.Mock
  const mockTrans: jest.Mock = Trans as jest.Mock
  beforeEach(() => {
    mockUseTranslation.mockReturnValue({
      t: (_ns: any, fallback: any) => fallback,
    })
    mockTrans.mockImplementation(({ i18nKey, children }) => children)
  })

  afterEach(() => {
    jest.clearAllMocks()
  })
  it('should render successfully', () => {
    const { baseElement } = render(<TerminologyIuu />)
    expect(baseElement).toMatchSnapshot()
  })
})
