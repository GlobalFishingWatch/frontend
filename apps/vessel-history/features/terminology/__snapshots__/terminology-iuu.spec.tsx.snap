// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TerminologyIuu should render successfully 1`] = `
<body>
  <div>
    <div
      class="container"
    >
      Illegal, unreported and unregulated (IUU) listing status according to official RFMO IUU vessel lists. A vessel is defined as “present on an RFMO blacklist” if the vessel is currently listed.” For more information, please see the
       
      <a
        href="https://drive.google.com/drive/folders/1njcl_iDFiaLALTT-bK_wdlil7ldIntU8"
      >
        FAQs
      </a>
      .
    </div>
  </div>
</body>
`;
